foundation_cms/static/scss/mixins.scss

```diff
@@ -0,0 +1,32 @@
+// ===================================
```

Comment:
Can we rename this file to `_mixins.scss`, please? My ultimate goal is to have the `_` prefix added to all partial SCSS files that are meant to be imported rather than compiled directly.